tests/cloud_playbooks/roles/packet-ci/tasks/delete-vms.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Check if temp directory for {{ test_name }} exists
+  stat:
+    path: "/tmp/{{ test_name }}"
+  register: temp_dir_details
+
+- name: "Cleanup temp directory for {{ test_name }}"
+  file:
+    path: "/tmp/{{ test_name }}"
+    state: absent
+  when:
+    - temp_dir_details.stat.exists
```

Comment:
If it didn't exist, this play would still work. You can also see later if the path was there or not by checking if this play has changed or not.

Reply:
you are right. copy paste error. i'll remove the when.